packages/client/modules/userDashboard/components/OrgBilling/OrgBillingUpgrade.tsx

```diff
@@ -39,17 +39,19 @@ const StyledPrimaryButton = styled(PrimaryButton)({
 
 interface Props {
   organization: OrgBillingUpgrade_organization
+  invoiceListRefetch?: (refetchVariables: {}) => void
 }
 
 const OrgBillingUpgrade = (props: Props) => {
-  const {organization} = props
+  const {organization, invoiceListRefetch} = props
   const {id: orgId, tier, orgUserCount} = organization
   const {activeUserCount} = orgUserCount
   const {togglePortal, closePortal, modalPortal} = useModal()
   return (
     <>
       {modalPortal(
         <CreditCardModal
+          invoiceListRefetch={invoiceListRefetch}
```

Comment:
A CC Modal shouldn't know (or care) about an invoice list. its job is to take in a number & upgrade an account.

instead of making it aware of the invoice list, try passing in a prop called `onUpgrade = () => invoiceListRefetch({orgId, first: 3})`. 
That'll keep it decoupled & dumb so it can be used in more places